airbyte-config/init/src/main/resources/seed/source_definitions.yaml

```diff
@@ -314,14 +314,14 @@
 - name: HubSpot
   sourceDefinitionId: 36c891d9-4bd9-43ac-bad2-10e12756272c
   dockerRepository: airbyte/source-hubspot
-  dockerImageTag: 0.1.41
+  dockerImageTag: 0.1.40
```

Comment:
the same

Reply:
thanks for noticing this. I didn't notice this when I merged master branch. fixed it